README.md

```diff
@@ -45,6 +45,7 @@ trunk check enable {linter}
 | C#              | [dotnet-format]                                                                                                      |
 | CircleCI Config | [circleci]                                                                                                           |
 | Cloudformation  | [cfnlint], [checkov]                                                                                                 |
+| CMake           | [cmake-format]                                                                                                       |
```

Comment:
Can you also add a markdown link to https://cmake-format.readthedocs.io/en/latest/ on line 100 :)

Reply:
Done!